sktime/transformations/series/boxcox.py

```diff
@@ -45,6 +45,20 @@ def __init__(self, bounds=None, method="mle", sp=None):
         super(BoxCoxTransformer, self).__init__()
 
     def fit(self, Z, X=None):
+        """Fit data.
+
+        Parameters
+        ----------
+        Z : pd.Series
+            Series to fit.
+        X : pd.DataFrame, optional (default=None)
+            Exogenous data used in transformation.
+
+        Returns
+        -------
+        Zt : pd.Series
```

Comment:
Fit always returns `self`

Reply:
Thanks, I didn't know this! Have changed this line.